v2/snmp/sessionfactory.go

```diff
@@ -94,20 +96,20 @@ func LoggingHooks(trace *ManagerTrace) ManagerOption {
 type SNMPVersion int
 
 const (
-	SNMPV1 SNMPVersion = iota
-	SNMPV2C
-	SNMPV3
+	SNMPV1  SNMPVersion = 0
+	SNMPV2C SNMPVersion = 1
```

Comment:
i think this should be 2, most people when talking about 2 mean 2c

Reply:
I think that's a protocol issue - has to be 1.